README.md

```diff
@@ -1,4 +1,95 @@
-# Rating Web Component 
+# Lit Based Rating Web Component for Virtual Labs
+----
+
+This repository contains the source code for the rating web component for virtual labs. The web component is written and packaged as a lit component, with some customisable parameters for the web-component.
+
+The rating component is further split into the following components:
+
+1. **`rating-display`** : This component has the display of the submitted rating as `stars `,it reads the data from the google sheet using the sheet API.
+
+2. **`rating-submit`** : This packs the Rate experiment button and the rating-modal, which could be placed on the experiment page and which is used for collecting the rating of the web component, and submits the rating to the google analytics, and gets stored into the google sheets.
+
+## Features 
+
+The following are the features of the rating web-component:
+
+- **rating-display** : 
+    - the `rating-display` component could be used separately for displaying the rating of the given experiment,
+    - the sheet api url could be specified as `experimentURL` alongside the given component as shown, 
+    example usage: 
+
+        `<rating-display experimentURL=<your sheet API url>></rating-display>`
```

Comment:
I am a little bit confused, why is this line still here? I thought we agreed to remove it

Reply:
It has already been removed, it does not show up in the README